substreams-helper/src/price.rs

```diff
@@ -0,0 +1,52 @@
+use std::ops::Div;
+use std::str::FromStr;
+
+use bigdecimal::BigDecimal;
+use serde::Deserialize;
+use substreams::Hex;
+
+use crate::{math, rpc};
+
+pub enum Network {
+    Ethereum,
+}
+
+#[derive(Deserialize)]
+struct Config {
+    ethereum: NetworkConfig,
+}
+
+#[derive(Deserialize)]
+struct NetworkConfig {
+    yearn_lens_oracle: String,
+    usdc_decimals: u8,
+}
+
+pub fn get_price(network: Network, token_address: &str) -> Result<BigDecimal, String> {
+    let config = Config {
+        ethereum: NetworkConfig {
+            yearn_lens_oracle: "83d95e0d5f402511db06817aff3f9ea88224b030".to_string(),
```

Comment:
Should make all addresses into named constants and put them in a module

Reply:
How about putting all config like this in a toml file?